scripts.py

```diff
@@ -1151,9 +1151,17 @@ def _integration(self, _db, id, name, protocol, goal):
             )
         integration = None
         if id:
-            integration = get_one(_db, ExternalIntegration, id==id)
-        if not integration and name:
+            integration = get_one(
+                _db, ExternalIntegration, ExternalIntegration.id==id
+            )
+            if not integration:
+                raise ValueError("No integration with ID %s." % id)
+        if name:
             integration = get_one(_db, ExternalIntegration, name=name)
+            if not integration and not (protocol and goal):
+                raise ValueError(
+                    'No integration with name "%s". To create it, you must also provide protocol and goal.' % name
+                )
```

Comment:
This is to give better error messages when I ran `configure_integration` with bad arguments.